testpilot/frontend/static-src/styles/modules/_experiment-details.scss

```diff
@@ -1,5 +1,18 @@
-.details-header-wrapper {
+.is-enabled {
```

Comment:
For some reason, when I use those rules, my text aligns the text to the left and top. Is that what you get?

Code:
```
.is-enabled {
  @include flex-container(row, center, center);
  background: $green;
  border-bottom: 1px solid $dark-green;
  color: $white;
  display: block;
  font-weight: 300;
  height: $grid-unit * 1.8;
  left: 0;
  padding-top: 1px;
  position: absolute;
  top: 0;
  width: 100%;
}
```
<img width="1116" alt="screen shot 2016-07-12 at 5 50 41 pm" src="https://cloud.githubusercontent.com/assets/9204835/16784948/4ef2ed38-4859-11e6-8eca-da54f06b8418.png">